src/partisan_hyparview_peer_service_manager.erl

```diff
@@ -462,7 +478,9 @@ handle_info({'EXIT', From, _Reason},
                           connections=Connections}};
 
 handle_info({connected, Peer, Tag, _RemoteState}, State) ->
-    handle_connect(Peer, Tag, State);
+    %% @todo retreive the PeerEpoch from the Peer.
```

Comment:
The information which a peer can provide is a set of the active view. In order to handle the join properly, we need to find a method to pass the epoch value to the `partisan_peer_service_server` (similar to the `get_local_state()` in the `partisan_peer_service_manager`).